web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -623,8 +652,14 @@ const ResourceCard = styled.div<{ hasAccess?: boolean }>`
 
   opacity: ${props => (props.hasAccess ? '1' : '0.45')};
 
-  &:hover {
+  &:hover,
+  &:focus-visible {
+    outline: none;
     background: ${props => props.theme.colors.spotBackground[1]};
+
+    ${ExternalLinkIconWrapper} {
```

Comment:
Thoughts on showing the icon all the time (same as `Guided` label)? When it shows up on hover it seems (at least to me) like something to click 😅